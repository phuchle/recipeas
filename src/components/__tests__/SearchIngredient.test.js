import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SearchIngredient from '../SearchIngredient';

describe('SearchIngredient', () => {
  test('renders without crashing', () => {
    const component = shallow(<SearchIngredient />);
    const tree = toJson(component);

    expect(tree).toMatchSnapshot();
  });
});
