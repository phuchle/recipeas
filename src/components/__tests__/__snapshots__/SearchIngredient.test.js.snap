// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchIngredient renders without crashing 1`] = `
<div>
  <h4>
    Search for an ingredient
  </h4>
  <form
    onSubmit={[Function]}
  >
    <InputGroup
      bsClass="input-group"
    >
      <FormControl
        bsClass="form-control"
        componentClass="input"
        onChange={[Function]}
        placeholder="Search"
        type="text"
        value=""
      />
      <InputGroupButton
        bsClass="input-group-btn"
      >
        <Button
          active={false}
          block={false}
          bsClass="btn"
          bsStyle="default"
          className="btn btn-default glyphicon glyphicon-search"
          disabled={false}
          type="submit"
        />
      </InputGroupButton>
    </InputGroup>
  </form>
  <Link
    replace={false}
    to="/modify-ingredients"
  >
    <Button
      active={false}
      block={true}
      bsClass="btn"
      bsSize="large"
      bsStyle="default"
      disabled={false}
      style={
        Object {
          "marginBottom": "10px",
          "marginTop": "10px",
        }
      }
    >
      Back
    </Button>
  </Link>
  <ListGroup
    bsClass="list-group"
  />
</div>
`;
